CHANGELOG.md

```diff
@@ -78,58 +78,58 @@ For the set of changes since the last stable release:
 
 ## 👨‍👩‍👧‍👦 Contributors
 
-| Contributor | Commits | Lines ± | Files Changed |
```

Comment:
@akaladarshi you need to fix your editor to stop making automatic changes like this, could you please reset the changelog to just make the one line change? otherwise this looks good to go!

Reply:
I have fixed it.